uber/templates/groups/history.html

```diff
@@ -30,14 +30,36 @@
 {% endfor %}
 </table>
 
+    {% if group.stripe_transactions %}
```

Comment:
Sure. As long as this conversation is here to remind us of the dependency it should be fine.